src/mongocrypt.h.in

```diff
@@ -328,6 +328,9 @@ mongocrypt_setopt_log_handler (mongocrypt_t *crypt,
 
 /**
  * Configure an AWS KMS provider on the @ref mongocrypt_t object.
+ * 
+ * This has been superseded by the more flexible:
```

Comment:
You should add a note about this deprecation and the other deprecations to the CHANGELOG.md

Reply:
Good point, done.